org-code-javabuilder/lib/src/main/java/dev/javabuilder/LocalMain.java

```diff
@@ -21,14 +21,22 @@ public static void main(String[] args) throws InterruptedException {
     // turn off the default console logger
     logger.setUseParentHandlers(false);
 
+    final LocalContentManager localContentManager;
```

Comment:
Side note -- do you use `LocalMain`? I don't know that I have, but the description of how to use it mentions `LocalProjectFileManager`, which no longer exists?

Since this is maybe the last cleanup PR for a while, is there a quick update you could add of how to use `LocalMain` to those docs if we are still using it?

https://github.com/code-dot-org/javabuilder/blob/main/org-code-javabuilder/README.md

Reply:
`LocalMain` was originally added so we could run Javabuilder without any connection to code-dot-org, which was useful in the very beginning of Javabuilder's life but I don't think has had usage in 6+ months. I think we might want to revisit this tech debt item :) https://codedotorg.atlassian.net/browse/JAVA-227 @jmkulwik do you have any thoughts here?

Reply:
Yeah, the above was my understanding as well. If we do end up keeping it around though, I have a documentation task I'm planning on picking up and I can add some detail there.